src/commands/moderation/ban.ts

```diff
@@ -61,7 +59,7 @@ export const ban: CommandDefinition = {
     },
 };
 
-function makeSuccessfulBanEmbed(user: UserLike, reason: string): discord.MessageEmbed {
+function makeSuccessfulBanEmbed(user: UserLike, reason: string): discord.EmbedBuilder {
```

Comment:
```suggestion
function makeSuccessfulBanEmbed(user: UserLike, reason: string): EmbedBuilder {
```
Can change all ```discod.``` prefixes and just import the individual modules themselves.